src/flexbox.rs

```diff
@@ -109,7 +109,7 @@ struct AlgoConstants {
 impl Taffy {
     /// Computes the layout of [`Taffy`] according to the flexbox algorithm
     pub(crate) fn compute(&mut self, root: Node, size: Size<Option<f32>>) {
-        let style = self.nodes[root].style;
+        let style = self.nodes[root].style.clone();
```

Comment:
I believe this should use a reference rather than a clone.

Reply:
I couldn't get that to work:

```
error[E0502]: cannot borrow `*self` as mutable because it is also borrowed as immutable
   --> src/flexbox.rs:119:30
    |
112 |         let style = &self.nodes[root].style;
    |                      ---------- immutable borrow occurs here
...
119 |             let first_pass = self.compute_preliminary(root, style.size.maybe_resolve(size), size, false, true);
    |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ mutable borrow occurs here
...
126 |                         .maybe_max(style.min_size.width.maybe_resolve(size.width))
    |                                    ---------------------------------------------- immutable borrow later used here

For more information about this error, try `rustc --explain E0502`.
error: could not compile `taffy` due to previous error
```

Reply:
At worst you should be able to clone just the individual properties you want (size, min_size, max_size) rather than the whole style struct. But I think you can avoid even that by pulling the maybe_resolve calculations up above the compute_preliminary calls and dropping the reference before making those calls.